docs/jwst/pixel_replace/main.rst

```diff
@@ -22,21 +26,21 @@ Adjacent Profile Approximation
 
 This is the default (and most extensively tested) algorithm for most spectroscopic modes.
 
-First, the input 2-d spectral cutout is scanned across the dispersion axis to determine
+First, the input 2-D spectral cutout is scanned across the dispersion axis to determine
 which cross-dispersion vectors (column or row, depending on dispersion direction) contain
 at least one flagged pixel. Next, for each affected vector, a median normalized profile is created.
 
-First, the adjacent arrays (the number of which is set by the step argument
+The adjacent arrays (the number of which is set by the step argument
 ``n_adjacent_cols``) are individually normalized. Next, each pixel in the profile is set to
 the median of the normalized values. This results in a median of normalized values filling the vector.
 
-Finally, this profile is scaled to the vector containing a missing pixel, and the value is
+Finally, this profile is scaled to the vector containing a missing pixel and the value is
```

Comment:
A compound sentence typically requests a comma, but the editor can have their way.

Reply:
Fixed in latest commit.